.gitignore

```diff
@@ -1,2 +1,3 @@
 .go-pkg-cache/
 report/
+tags
```

Comment:
I'm not quite sure what `tags` are or why we're supposed to be ignoring them. Is this something new to the singular repo?

Reply:
Oh not new to any repo or anything, this is the file generated by ctags (I use it to browse the python code). I figured there was no harm in adding it to .gitignore, but if you think that could have negative side effects I can just drop it...

Reply:
I don't think it has any negative side effects but I'm a little hesitant to add things that are unique to our individual dev environments. We _should_ be fine as long as new files aren't added with `git add .` so I don't think there is necessarily a reason to keep this either.

Reply:
Fair point, will drop this change. Thanks @mgleung!